drivers/public_reports/app/models/public_reports/homeless_population.rb

```diff
@@ -438,6 +439,29 @@ def populations
       }.freeze
     end
 
+    private def ethnicity_chart(population)
+      {}.tap do |charts|
+        quarter_dates.each do |date|
+          charts[date.iso8601] = {
+            data: with_service_in_quarter(report_scope, date, population).
+              joins(:client).
+              group(c_t[:Ethnicity]).
+              count.
+              map do |e_id, count|
+                # Force any unknown genders to Unknown
```

Comment:
This comment confused me -- but I think it is just a copy/paste error... Should be unknown ethnicities?